HiLo.java

```diff
@@ -0,0 +1,65 @@
+import java.util.Scanner;
+
+public class HiLo {
+   public static void main(String[] args) {
+      Scanner sc = new Scanner(System.in);
```

Comment:
Generally it is a good practice keeping your variables `final` wherever you can. This would be good example. That way, you are making them immutable, which is preferable state for variables.